packages/blockchain-wallet-v4-frontend/src/modals/OrderMyCard/model.tsx

```diff
@@ -28,9 +28,33 @@ const StyledSeeCardDetailsButton = styled(Button)`
   }
 `
 
-const StyledButton = styled(Button)`
-  margin-top: 32px;
+export const StyledButton = styled(Button)`
   width: 327px;
+  margin-bottom: 8px;
+`
+
+export const ResultWrapper = styled.div`
+  width: 327px;
+  text-align: center;
+  margin-top: -10%;
+`
+
+export const ResultTitleWrapper = styled(Text)`
+  font-size: 1.25rem;
+  color: black;
+  font-weight: 600;
+  margin-bottom: 8px;
+`
+
+export const ResultSubTitleWrapper = styled(Text)`
+  font-size: 0.875rem;
+  color: ${(props) => props.theme.grey600};
+  font-weight: 500;
+  margin-bottom: 24px;
```

Comment:
rem?

Reply:
In other PR where I put rem I was asked to use px, while there is no clear definition about rem or px in margins I would prefer to keep it simple with px